docs/custom_rules.md

```diff
@@ -0,0 +1,78 @@
+# Custom Rules
+The Custom Rules feature allows any customer to create simple rules using a pre-defined set of operators. These rules support any resource property comparisons.
+
+# Syntax
+Each rule has been designed to be easy to write, and thus the syntax is very flexible in regards to grammar. Each rule is always one line long, and follows a set structure in regards to syntax.
+
+The template used for each custom rules have been included below. Angle brackets specify required values, while square brackets indicate optional values.
+
+`<Resource Type> <Property[*]> <Operator> <Value> [Error Level] [Custom Error Message]`
+
+
+### General Guidelines
+
+* The ruleID is auto-generated based on the line number. The E9XXX and W9XXX blocks are allocated towards custom rules.
+    * As an example, custom rule on line 4 of the rules file which has error level “ERROR” would become E9004
+* Comments are supported through the use of the # symbol at the beginning of a line (e.g `#This is a comment`)
+* The syntax is "quote-flexible" and will support all permutations shown below
+```
+AWS::EC2::Instance Property EQUALS "Cloud Formation"
+AWS::EC2::Instance Property EQUALS Cloud Formation
+AWS::EC2::Instance Property EQUALS Cloud Formation WARN "Custom Error"
+AWS::EC2::Instance Property EQUALS Cloud Formation WARN Custom Error
+```
+
+#### Resource Type
+
+Name of the resource type as specified within the template. (e.g `AWS::EC2::Instance`)
+
+
+#### Property
+
+Any property of a resource. Dot notation may be used to specify lower-level properties. (e.g `AssumeRolePolicyDocument.Version`)
+
+#### Operator
+
+The specified operator to be used for this rule. The supported values are defined [here](#supported-operators)
```

Comment:
Why is the table not here?

Reply:
Fair point. I thought it would look messy if I put it in the middle of the list of all syntax terms, but it actually looks okay there.

Moved!